cucumber/authenticators_gcp/features/step_definitions/authn_gcp_steps.rb

```diff
@@ -0,0 +1,23 @@
+
+Given(/^I set (service-account-id) annotation (with incorrect value )?to host "([^"]*)"$/) do |annotation_name, incorrect_value, hostname|
+    i_have_a_resource "host", hostname
+    case annotation_name
+    when "service-account-id"
+      annotation_value = gcp_service_account_id
+    else
+      raise "incorrect annotation name #{annotation_name}"
+    end
+
+    set_annotation_to_resource("authn-azure/#{annotation_name}", annotation_value)
```

Comment:
we can move the `authn-azure/` prefix to the annotations name in the input so it is shown also in the feature file (less voodoo). this will also let us move this helper to a more general file (maybe `user_steps.rb`?)